### DISCUSSION TITLE
TALL : How to apply a js script on an element after the render and only once

### DISCUSSION THREAD

**Discussion by dah-HUB** [livewire/livewire#5333] [1. Help]
Hi all,
When a Livewire component Is loaded inside a Modal or a slide-over, things like select2, tom-select .... are not working.

Q : How to apply, for example :  <script>new TomSelect("#users",{});</script>
after the render and only once ?

the blade looks like :
div wire:ignore
    select id="users" wire:model.defer="users" class="tom-select" multiple
        @foreach($user_list as $key => $value)
            option value="{{ $key }}"> {{ $value }} /option
        @endforeach
    /select
/div

Q : Do you know any free livewire slide-over component ?

**Comment by joshhanley** [comment]
@DAH-hub Livewire can't dynamically load scripts on subsequent renders. So generally I try to make sure my modal/slideover component already exists on the page but is hidden, that way all inputs that require scripts are loaded on the initial request.

Hope this helps!
